### PR TITLE
Fix Logout button

### PR DESCRIPTION
**Pull request by rotimi-babalola**
## What does this PR do?
Fixes logout button bug. Clicking the logout button now redirects to `login` page and removes the token from local storage. 

## Description of Task to be completed?
Fix issue with Logout Button

## How should this be manually tested?
Click the logout button. You should not able to access the protected routes unless you log in again

## What are the relevant pivotal tracker stories?
#158288797

## Any background context you want to add?
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
- [x] Raise PR
- [ ] Test
- [ ] Documentation
## Screenshots

### CODE REVIEW

File: src/_components/NavBarComponent.jsx [hunk 1]
```diff
@@ -1,35 +1,62 @@
...
-export default NavBarComponent;
+import React, { Component } from 'react';
+import jwt from 'jsonwebtoken';
+import { withRouter } from 'react-router-dom';
+import { Dropdown, Menu, Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import '../_css/NavBarComponent.css';
+
+/* eslint require-jsdoc: 0 */
```
Comment (theterminalguy) [inline src/_components/NavBarComponent.jsx]: Please get rid of all this comment. We won't need once we fix the JSLint configuration

**Comment by theterminalguy** [comment]
There are too many changes here, please revert back to the original file structure and ignore all JSLINT warnings from hound ci. I only expect to see the logic for logging out 

@rotimi-babalola

**Comment by rotimi-babalola** [comment]
I've fixed the logic now @theterminalguy

**Comment by sachungo** [comment]
@rotimi-babalola the token is being renamed to `art-prod-web-token` in https://github.com/AndelaOSP/art-dashboard/pull/36

**Pull request closed without merging**